src/pipecat/serializers/TwilioFrameSerializer.py

```diff
@@ -0,0 +1,58 @@
+import audioop
+import base64
+import json
+
+from pipecat.frames.frames import AudioRawFrame, Frame
+from pipecat.serializers.base_serializer import FrameSerializer
+
+
+class TwilioFrameSerializer(FrameSerializer):
+    SERIALIZABLE_TYPES = {
+        AudioRawFrame: "audio",
+    }
+
+    def __init__(self):
+        self.sid = None
+
+    def ulaw_8000_to_pcm_16000(self, ulaw_8000_bytes):
+        audio_data = base64.b64decode(ulaw_8000_bytes)
+
+        # Convert μ-law to PCM
+        pcm_8000_bytes = audioop.ulaw2lin(audio_data, 2)
+
+        # Resample from 8000 Hz to 16000 Hz
+        pcm_16000_bytes = audioop.ratecv(pcm_8000_bytes, 2, 1, 8000, 16000, None)[0]
+
+        return pcm_16000_bytes
+
+    def pcm_16000_to_ulaw_8000(self, pcm_16000_bytes):
+        # Resample from 16000 Hz to 8000 Hz
+        pcm_8000_bytes = audioop.ratecv(pcm_16000_bytes, 2, 1, 16000, 8000, None)[0]
+
+        # Convert PCM to μ-law
+        ulaw_8000_bytes = audioop.lin2ulaw(pcm_8000_bytes, 2)
+
+        return ulaw_8000_bytes
```

Comment:
Can we move `pcm_16000_to_ulaw_8000` and `ulaw_8000_to_pcm_16000` to `src/pipecat/utils/audio.py`? This way it could be easily used by others in the future.

Reply:
Yep